tables/Tables/TableUtil.cc

```diff
@@ -137,28 +144,28 @@ namespace casacore {
                          Bool checkSubTables, Bool splitColons)
     {
       if (splitColons) {
-        std::pair<Table,String> t = tryFindSubTable (tableName, True);
+        std::pair<Table,String> t = findParentTable (tableName);
         if (! (t.first.isNull()  ||  t.second.empty())) {
```

Comment:
Only checking `!t.first.isNull()` should be enough.

Reply:
Your're right. Changed at all places.